ontology/nomen.md

```diff
@@ -14,8 +14,8 @@ contact:
 description: NOMEN is a nomenclatural ontology for biological names (not concepts).  It encodes the goverened rules of nomenclature.
 domain: information
 funded_by:
-- NSF-ABI-1356381
-- Species File Group
```

Comment:
note that `Species File Group` does not contain any substantial information, other than the name of the group that developed the ontology, and is not itself a funding source the way this field is expected to be used.

Reply:
The Species File Group has an endowment and so may be a funding source. @mjy should review.

Reply:
Oops I see Matt already responded this morning—never mind!